components/menu/stories/menu.test.js

```diff
@@ -196,23 +218,44 @@ export const MenuItemGroup = Variants({
 			testHeading: "Disabled",
 			isDisabled: true,
 		},
-		// {
-		// 	testHeading: "Selected",
-		// 	isSelected: true,
-		// },
-		// {
-		// 	testHeading: "Selected and disabled",
-		// 	isSelected: true,
-		// 	isDisabled: true,
-		// },
-		// {
-		// 	testHeading: "Collapsible",
-		// 	isCollapsible: true,
-		// },
-		// {
-		// 	testHeading: "Drill-in",
-		// 	isDrillIn: true,
-		// },
+		{
+			testHeading: "Without icon",
+			iconName: undefined,
+		},
+		{
+			testHeading: "Without value",
+			value: undefined,
+			include: ["No selection", "No selection, with description", "Multi-selection: unselected", "Multi-selection: switches + labels", "Drill-in"],
+		},
+		{
+			testHeading: "Without value or icon",
+			iconName: undefined,
+			value: undefined,
+			include: ["No selection", "No selection, with description", "Multi-selection: unselected", "Multi-selection: switches + labels", "Drill-in"],
+		},
+		{
+			testHeading: "With value",
+			value: "⌘ N",
+			include: ["Truncation", "Text wrapping"],
+		},
+		{
+			testHeading: "With multi-select switch",
+			selectionMode: "multiple",
+			hasActions: true,
+			include: ["Truncation", "Text wrapping"],
+		},
+		{
+			testHeading: "With value and multi-select switch",
+			selectionMode: "multiple",
+			hasActions: true,
+			value: "⌘ N",
+			include: ["Truncation", "Text wrapping"],
+		},
+		{
+			testHeading: "Without description",
+			description: undefined,
+			include: ["Drill-in", "Truncation", "Text wrapping"],
+		},
 		// {
 		// 	testHeading: "Highlighted",
 		// 	isHighlighted: true,
```

Comment:
What is this `isHighlighted`? Maybe that's like...focused? Or selected?

Reply:
I wasn't sure on this one either, so I left it be.